foreman_ansible.gemspec

```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'acts_as_list', '~> 1.0.3'
   s.add_dependency 'deface', '< 2.0'
-  s.add_dependency 'foreman_remote_execution', '>= 8.0.0'
-  s.add_dependency 'foreman-tasks', '>= 7.0.0'
+  s.add_dependency 'foreman_remote_execution', '~> 8.0.0'
```

Comment:
This pins to 8.0.x. Did you intend 8.x?
```suggestion
  s.add_dependency 'foreman_remote_execution', '~> 8.0'
```

Same below for foreman-tasks.

Reply:
Yes, thank you @ekohl!